### PR TITLE
Chromium, Google Chrome: update to 126.0.6478.182

### PR DESCRIPTION
**Pull request by MingcongBai**
Topic Description
-----------------

- google-chrome: update to 126.0.6478.182
- chromium: update to 126.0.6478.182

Package(s) Affected
-------------------

- chromium: 126.0.6478.182
- google-chrome: 126.0.6478.182

Security Update?
----------------

No

Build Order
-----------

```
#buildit chromium google-chrome
```

Test Build(s) Done
------------------

**Primary Architectures**

- [x] AMD64 `amd64`
- [x] AArch64 `arm64`
- [x] LoongArch 64-bit `loongarch64`

### CODE REVIEW

**Comment by jiegec** [comment]
缺一个 patch：https://salsa.debian.org/chromium-team/chromium/-/blob/master/debian/patches/upstream/blink-header2.patch?ref_type=heads

**Comment by MingcongBai** [comment]
> 缺一个 patch：https://salsa.debian.org/chromium-team/chromium/-/blob/master/debian/patches/upstream/blink-header2.patch?ref_type=heads

多谢，我重试下

**Comment by Cyanoxygen** [comment]
Tested on amd64.
image

**Review by MingcongBai — COMMENTED** [review]
Tested good on LoongArch: 

图片

File: app-web/chromium/autobuild/patches/wget-log [hunk 1]
```diff
@@ -0,0 +1,11 @@
+--2024-07-18 11:14:52--  https://salsa.debian.org/chromium-team/chromium/-/raw/master/debian/patches/upstream/blink-header2.patch?ref_type=heads
```
Comment (Cyanoxygen) [inline app-web/chromium/autobuild/patches/wget-log]: Wait! you can not put `wget-log` in the source tree. It is a log file.

**Review by Cyanoxygen — CHANGES_REQUESTED** [review]

**Review by Cyanoxygen — APPROVED** [review]

**Pull request merged**